src/app/(Authentication)/SignUp/Address/index.tsx

```diff
@@ -122,28 +123,21 @@ export default function SignUpScreen() {
       </View>
       <Text style={styles.space}> </Text>
 
-      <TouchableOpacity
+      <ButtonBlack
         disabled={
           streetAddress.trim() === '' ||
           city.trim() === '' ||
           state.trim() === '' ||
           zipcode.trim() === ''
         }
-        style={
-          streetAddress.trim() === '' ||
-          city.trim() === '' ||
-          state.trim() === '' ||
-          zipcode.trim() === ''
-            ? styles.nextButtonGray
-            : styles.nextButton
-        }
+        style={styles.nextButton}
         onPress={() => handleSubmit()}
       >
         <Text style={styles.nextText}>Sign Up</Text>
         <View style={styles.check}>
           <Check />
         </View>
```

Comment:
can we remove the view that wraps the check icon. Instead wrap both the "Sign Up" text and icon with one View container that centers them along the same axis and separates them by a column gap. That way we can remove margins from text components.